js/data/security/htmlFilter.js

```diff
@@ -0,0 +1,19 @@
+export const htmlFilter = {
```

Comment:
Since we haven't decided what (if any) type of sanitization we want to do here and since we're leaning towards none being necessary, can we remove any sanitization related code? I'd rather not put code into the code-base that -might- be necessary.